src/AddImageChecks.cpp

```diff
@@ -478,8 +479,24 @@ Stmt add_image_checks(Stmt s,
             // Check the var passed in equals the constrained version (when not in inference mode)
             asserts_constrained.push_back(AssertStmt::make(var == constrained_var, error));
         }
+        if (param.defined() && param.host_alignment() != param.type().bytes()) {
+            string host_name = name + ".host";
+            int alignment_required = param.host_alignment();
+            Expr host_ptr = Variable::make(Int(32), host_name);
```

Comment:
Thanks. I'll do that.
@abadams : Thanks for the tip. I see that now in lesson 14 ;)